src/components/ResourceList/components/BulkActions/components/BulkActionButton/BulkActionButton.tsx

```diff
@@ -20,14 +19,13 @@ export default class BulkActionButton extends React.PureComponent<
   Props,
   never
 > {
+  private bulkActionButton = createRef<HTMLButtonElement>();
+
   componentDidMount() {
     const {handleMeasurement} = this.props;
     if (handleMeasurement) {
-      const bulkActionButtonNode = findDOMNode(this);
-      const width =
-        (bulkActionButtonNode instanceof Element &&
-          bulkActionButtonNode.getBoundingClientRect().width) ||
-        0;
+      const width = (this.bulkActionButton
+        .current as HTMLButtonElement).getBoundingClientRect().width;
```

Comment:
This cast is to increase codecov. We don't optionally render the button so this ref will always exist in componentDidMount lifecycle